daemon/containerd/image_tag.go

```diff
@@ -60,7 +60,22 @@ func (i *ImageService) TagImageWithReference(ctx context.Context, imageID image.
 			return errdefs.System(err)
 		}
 	}
-
 	logger.Info("image created")
+
+	if err == nil {
```

Comment:
Looks like `err != nil` is already handled at line 34 above;

```go
	_, err = is.Create(ctx, newImg)
	if err != nil {
```

Reply:
Good catch, thanks!